### PR TITLE
Install Kafka UI together with Strimzi

### PR DESCRIPTION
**Pull request by mgencur**
* allows for inspecting existing Topics and messages and seeing if all messages were delivered or still hanging in Kafka

More info about the UI at https://github.com/provectus/kafka-ui

### CODE REVIEW

File: hack/lib/strimzi.bash [hunk 1]
```diff
@@ -134,11 +134,49 @@ EOF
...
+  labels:
+    app: kafka-ui
+spec:
+  containers:
+    - env:
+      - name: KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS
+        value: my-cluster-kafka-bootstrap.kafka.svc:9092
+      - name: KAFKA_CLUSTERS_0_NAME
+        value: my-cluster
+      image: docker.io/provectuslabs/kafka-ui
```
Comment (matzew) [inline hack/lib/strimzi.bash:155]: will this get same "rate limit" issues we had sometime ago w/ maistra? Or is their accoount a "better" one? :see_no_evil:

Comment (matzew) [reply hack/lib/strimzi.bash:155]: I guess we can't do much, when they publish to dockerhub... :-)

**Review by matzew — APPROVED** [review]
/lgtm
/approve

File: hack/lib/strimzi.bash (hunk 1 above)
Comment (mgencur) [reply hack/lib/strimzi.bash:155]: We could probably mirror this image. Maybe I should push it to quay?

Comment (mgencur) [reply hack/lib/strimzi.bash:155]: Pushing to quay. Will update this PR when it's ready

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**